monitoring/harmony-monitor/blockchain-watchdog/root.go

```diff
@@ -85,7 +85,7 @@ func (service *Service) compareIPInShardFileWithNodes() error {
 
 	for shardID, subcommittee := range service.superCommittee {
 		results.Store(shardID, hooligans{})
-		for _, nodeIP := range subcommittee {
+		for _, nodeIP := range subcommittee.members {
 			wait.Add(1)
 			go func(shardID int, nodeIP string) {
 				defer wait.Add(-1)
```

Comment:
Why use wait.Add(-1) instead of wait.Done()?

Reply:
its the same thing but I prefer .Add(-1) as is explicit on decrement